src/d3d9/d3d9_device.cpp

```diff
@@ -7813,6 +7816,7 @@ namespace dxvk {
     }
     else
       m_implicitSwapchain = new D3D9SwapChainEx(this, pPresentationParameters, pFullscreenDisplayMode);
+      m_mostRecentlyUsedSwapchain = m_implicitSwapchain.ptr();
```

Comment:
There's `ResetLastUsedSwapchain()` for this now.

Reply:
I'm not a fan of calling that function for the initial initialization.